Library/Formula/g3log.rb

```diff
@@ -0,0 +1,34 @@
+class G3log < Formula
+  desc "asynchronous, \"crash safe\", logger that is easy to use."
```

Comment:
Yes. I guess it's a matter of taste to escape or switch to single quotes, since you use doubles quotes I thought you would prefer them escaped. Anything else you would like changed while I'm at it? :)